volatility3/framework/symbols/linux/__init__.py

```diff
@@ -267,3 +267,12 @@ def walk_internal_list(cls, vmlinux, struct_name, list_member, list_start):
             list_struct = vmlinux.object(object_type = struct_name, offset = list_start.vol.offset)
             yield list_struct
             list_start = getattr(list_struct, list_member)
+
+    @classmethod
+    def container_of(cls, addr, type_name, member_name, vmlinux):
```

Comment:
this function seems useful, but I do not see it used anywhere in the rest of the commit. I just wanted to make sure this was meant to be part of this pull request.

Reply:
Good catch! It was used before the last commit changes #[192759b](https://github.com/volatilityfoundation/volatility3/pull/600/commits/192759b5d5c8c2622059847fbd915faf8af66e50). 
I just removed it as it is also included in the SockStats PR. I mentioned this in the "merge note" above.